x-pack/plugins/apm/public/components/shared/unified_search_bar/index.tsx

```diff
@@ -183,7 +223,32 @@ export function UnifiedSearchBar({
       query: filter,
     } as Filter;
   });
-  const handleSubmit = (payload: { dateRange: TimeRange; query?: Query }) => {
+
+  const onRefresh = () => {
+    clearCache();
+    incrementTimeRangeId();
+  };
```

Comment:
was this never implemented in the unified search bar implementation?

Reply:
Do you mean on the original Unified Search Component from Platform ?
On our side, this was a miss which caused the refresh button to not work